app/services/workers/calculate-workload-points.js

```diff
@@ -17,27 +17,26 @@ const wpcOperationType = require('../../constants/calculate-workload-points-oper
 const adjustmentCategory = require('../../constants/adjustment-category')
 
 module.exports.execute = function (task) {
-  var id = task.additionalData.workloadBatch.startingId
+  var startingStagingId = task.additionalData.workloadBatch.startingId
   var batchSize = task.additionalData.workloadBatch.batchSize
   var reportId = task.workloadReportId
   var operationType = task.additionalData.operationType
-  var maxId = id
+  var maxStagingId = startingStagingId + batchSize - 1
   var message
 
   if (batchSize <= 0) {
     logger.error('Batchsize must be greater than 0')
     throw (new Error('Batchsize must be greater than 0'))
   } else if (batchSize > 1) {
-    maxId = id + batchSize - 1
-    message = 'Calculating Workload Points for Workloads ' + id + ' - ' + (id + batchSize - 1)
+    message = 'Calculating Workload Points for workloads with staging ids ' + startingStagingId + ' - ' + (startingStagingId + batchSize - 1)
```

Comment:
No need to recalculate the upper bound here, we can just reuse the `maxStagingId` variable.